js/main.js

```diff
@@ -172,10 +172,6 @@ const displayPopupModal = (i) => {
 };
 document.querySelectorAll('.see-project').forEach((row) => row.addEventListener('click', () => displayPopupModal(row.getAttribute('data-index'))));
```

Comment:
- [ ] You did a good job with the functionalities but for this project, you are supposed to stick to the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) design as much as possible. Please increase the size of the close(X) button and reduce the mobile menu options and make sure it matches the template.

###  Your design

![image](https://user-images.githubusercontent.com/83803414/167693314-f991d7c5-2d1a-4173-add7-cc9340f68d0a.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/167693489-e56937ad-6627-4363-94ac-cc6bcc48ab1f.png)
![image](https://user-images.githubusercontent.com/83803414/167693489-e56937ad-6627-4363-94ac-cc6bcc48ab1f.png)